Class/practice3.2/Triangle.cpp

```diff
@@ -0,0 +1,33 @@
+#include "UnitTest.hpp"
+#include <cmath>
+#include <functional> // Add this for std::function
+#include <vector>     // Add this for std::vector
+
+class Triangle {
+public:
+    Triangle(double a, double b, double c) : sideA(a), sideB(b), sideC(c) {}
+
+    double calculateArea() const {
+        // Heron's formula to calculate the area of a triangle
+        double s = (sideA + sideB + sideC) / 2;
+        return std::sqrt(s * (s - sideA) * (s - sideB) * (s - sideC));
```

Comment:
not every triple of double are form triangle - we should check inequality of triangle and the fact that values are positive